src/charts/utilization-bar/utilization-bar-chart.html

```diff
@@ -0,0 +1,34 @@
+<div class="pf-utilization-bar-chart">
+  <span ng-if="!layout || layout.type === 'regular'">
+    <div ng-if="chartTitle" class="progress-description">{{chartTitle}}</div>
+    <div class="progress progress-label-top-right">
+      <div class="progress-bar" role="progressbar" ng-class="{'animate': animate,
+           'progress-bar-success': isOk, 'progress-bar-danger': isError, 'progress-bar-warning': isWarn}"
+           style="width:{{chartData.percentageUsed}}%" tooltip="{{chartData.percentageUsed}}% Used">
+        <span ng-if="!usedLabelFormat || usedLabelFormat === 'actual'">{{chartData.used}} of {{chartData.total}} {{units}} Used</span>
+        <span ng-if="usedLabelFormat === 'percent'">{{chartData.percentageUsed}}% Used</span>
+      </div>
+      <div class="progress-bar progress-bar-remaining" role="progressbar" aria-valuenow="5" aria-valuemin="0" aria-valuemax="100"
+           style="width: {{100-chartData.percentageUsed}}%" tooltip="{{100-chartData.percentageUsed}}% Available">
+      </div>
+    </div>
+  </span>
+  <span ng-if="layout && layout.type === 'inline'">
```

Comment:
I merged in the CSS. Can you please update this PR to look at patternfly master.